test/benchmarks/perftest.pl

```diff
@@ -85,8 +86,8 @@
 
 if(!$is_baseline)
 {
-    open(IN,"$basefile") or die "Couldn't open $basefile for reading.";
-    while(<IN>)
+    open my $IN, '<', $basefile or die "Couldn't open $basefile for reading.";
```

Comment:
Here and elsewhere I think I prefer the parens, unless removing them is actually part of strict mode requirements.